### ISSUE TITLE
EPUB Validation error with Sphinx 1.6.1

### ISSUE THREAD

**Issue by juh2** [sphinx-doc/sphinx#3763]
Subject: I get a lot of validation errors when validating an epub with epubcheck 4.0.2

### Problem

Sphinx does not produce valid epub.

#### Procedure to reproduce the problem
```
Run make epub on any project and validate the epub with epubcheck 4.0.2

```

#### Error logs / results
```



```
- <public link of unexpected result if you have>

#### Expected results
<Describe what to actually do>

### Reproducible project / your project
- <link to your project, or attach zipped small project sample>

### Environment info
- OS: Linux  4.8.0-52-generic #55-Ubuntu SMP Fri Apr 28 13:28:50 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
- Python version: 3.5.2
- Sphinx version: 1.6.1
- <Extra tools e.g.: Browser, tex or something else>

**Comment by tk0miya** [comment]
It might have been fixed at #3756.
Could you check it please? Or please share your document if possible.

**Comment by tk0miya** [comment]
I got reproducible project directly.
And fixed at #3756 and 2387d51.

Thank you for reporting!
